docs/pages/repo/docs/reference/command-line-reference/run.mdx

```diff
@@ -224,6 +224,32 @@ Just a quick overview.
 - `{}` allows for a comma-separated list of "or" expressions
 - `!` at the beginning of a pattern will negate the match
 
+### `--log-order`
+
+`type: string`
+
+Set type of task output order. Defaults to "auto" to let
+turbo decide based on its own heuristics.
```

Comment:
We're mentioning "using heuristics" but I'm left wondering what _are_ those heuristics? What behaviors should I be expecting to see?